### DISCUSSION TITLE
Remove the Forum menu entry ?

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#950] [Ideas]
Since the [Sourceforge forum is discontinued](https://sourceforge.net/p/mytourbook/discussion/622811/thread/10e08732d0/), maybe that menu entry should be removed ? or replaced by the [GitHub Issues](https://github.com/mytourbook/mytourbook/discussions) link ?

![image](https://user-images.githubusercontent.com/25235665/197316308-393f6511-73bc-4d0b-8ae1-296781904e7f.png)

**Comment by wolfgang-ch** [comment]
Issue created https://github.com/mytourbook/mytourbook/issues/951
